Spanner/src/Connection/Grpc.php

```diff
@@ -1092,4 +1167,71 @@ private function getDatabaseAdminClient()
 
         return $this->databaseAdminClient;
     }
+
+    /**
+     * Allow lazy instantiation of the spanner client.
+     * If routing is enabled, a SpannerClient is returned for a specific instanceName
+     * with a binding in the connection of a specific endpoint.
+     * If the endpoint list is not requested, a request is first made for this instanceName.
+     *
+     * @param string $instanceName Optional. Full name of instance.
+     * @return SpannerClient
+     */
+    private function getSpannerClient($instanceName = null)
+    {
+        if (!$this->spannerClient) {
+            $this->spannerClient = $this->constructGapic(SpannerClient::class, $this->grpcConfig);
+        }
+
+        if ($this->enableResourceCaching) {
+            if (isset($this->spannerClients[$instanceName])) {
+                return $this->spannerClients[$instanceName];
+            } elseif (!isEmpty($instanceName) && !isset($this->endpointUris[$instanceName])) {
+                $parsed = InstanceAdminClient::parseName($instanceName);
+                $projectId = $parsed['project'];
```

Comment:
This could be shortened to 
```php
$projectId = InstanceAdminClient::parseName($instanceName)['project'];
```
It seems `$parsed` is not used anywhere else in this scope.

Reply:
ok